hooks/pre-command

```diff
@@ -29,7 +29,7 @@ compute_tag() {
   local docker_file="$1"
   local sums=($(sha1sum "${docker_file}"))
   while IFS='=' read -r name _ ; do
-    if [[ $name =~ ^(BUILDKITE_PLUGIN_DOCKER_ECR_CACHE_CACHE_ON_[0-9]+) ]] ; then
+    if [[ $name =~ ^(BUILDKITE_PLUGIN_DOCKER_ECR_PUBLISH_CACHE_ON_[0-9]+) ]] ; then
```

Comment:
Does the caching logic still make sense? If we're building and pushing application images with this plugin, we would `cache-on` basically every source file.

Reply:
I don't know that I follow? My understanding is that the caching logic means that we'll only push an image if the dockerfile changes, which seems useful to me, particularly in the case of ca-docker-images where generally 2/3 won't have changed.